src/jit/lclvars.cpp

```diff
@@ -6491,6 +6601,14 @@ void Compiler::lvaDumpEntry(unsigned lclNum, FrameLayoutState curState, size_t r
     {
         printf(" stack-byref");
     }
+    if (varDsc->lvClassHnd != nullptr)
+    {
+        printf(" class-hnd");
```

Comment:
Class names get pretty long and there should now be notes left earlier in the dump stream when this field is set that correlate the variable, the name of the type, and the corresponding handle address.  So my instinct is to leave this as is.

I could add this as a dump option, though we seem to have perhaps too many dump options already.